MatrixKit/Models/Room/MXKRoomDataSource.m

```diff
@@ -2561,14 +2561,7 @@ - (void)processQueuedEvents:(void (^)(NSUInteger addedHistoryCellNb, NSUInteger
 
 #pragma mark - UITableViewDataSource
 - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section
-{
-    // The view controller is going to display all messages
-    // Automatically reset the counters
-    if ([UIApplication sharedApplication].applicationState == UIApplicationStateActive)
-    {
-        [self markAllAsRead];
-    }
-    
+{    
```

Comment:
Without this piece of code, the app never sends read receipt on messages that come in the room the user is reading